docker-java-api/src/main/java/com/github/dockerjava/api/model/Ulimit.java

```diff
@@ -39,11 +46,23 @@ public String getName() {
         return name;
     }
 
+    @JsonIgnore
```

Comment:
let's deprecate getters that return `Integer`

Reply:
Done